addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -156,6 +156,7 @@
             <script type="text/javascript" src="/web/static/src/js/public/public_widget.js"/>
             <script type="text/javascript" src="/web/static/src/js/services/ajax_service.js"></script>
             <script type="text/javascript" src="/web/static/src/js/services/local_storage_service.js"></script>
+            <script type="text/javascript" src="/web/static/src/js/public/lazyloader.js"/>
```

Comment:
Maybe place it before "public_root.js" as it's required by "public_root_instance.js" ?

Just to have some kind of logic within the imports.